src/components/NoMessageSelected.vue

```diff
@@ -52,7 +52,7 @@ export default {
 }
 .app-content {
 	display: flex;
-	height: 100%;
+	height: 100% !important;
```

Comment:
is there a conflicting rule that set a different height?

Reply:
yes, the app-content-details, that breaks the message content.

https://github.com/nextcloud/mail/pull/10571/files#diff-2dc54a20fa4ab6845b4b0f252336e107d33a13f5c05d966e5c104770a1b292fdR204

Reply:
Wasn't that just added with https://github.com/nextcloud/mail/commit/8089ccb0f3915d57520458ecf0ce8729341cfbd4?

This doesn't smell like a maintainable solution if the v-deep is leaking to places it shouldn't and we have to pull the `!important` overwrite to push back.

Would it be possible to get rid of the deep selector?

Reply:
Is Contacts missing `scoped` for the component styles?

Reply:
it comes from a css file: https://github.com/nextcloud/contacts/blob/main/src/css/contacts.scss#L25
I dont see any vue component on contacts not to be scoped and has this class

Reply:
Can we move the css file into the app component that uses the lib component that is being styled?